spec/features/viewing_peeps_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature 'Viewing peeps' do
+  scenario 'A user can see peeps in reverse chronological order' do
+    connection = PG.connect(dbname: 'chitter_test')
+
+    connection.exec("INSERT INTO peeps VALUES(1, 'Good morning');")
```

Comment:
If the 1 is the peep/message_id i do not think you need to insert it if the column is a SERIAL PRIMARY KEY, it will do that automatically for you.